src/components/NavBar.astro

```diff
@@ -0,0 +1,15 @@
+---
+import { Icon } from "astro-icon";
+---
+
+<div class="flex justify-end">
+  <ul
+    class="top-4 right-4 text-center flex font-semibold gap-16 p-4 items-center"
+  >
+    <li><a href="http://google.com">Invest</a></li>
+    <li><a href="http://google.com">Insure</a></li>
+    <li><a href="http://google.com">Goals</a></li>
+    <li><a href="http://google.com">Partners</a></li>
+    <button><Icon class="h-3.5" name={"bars"} /></button>
```

Comment:
The curly brackets are not needed because you are printing an string.
 
```suggestion
    <button><Icon class="h-3.5" name="bars" /></button>
```

Reply:
I made the requested changes